site/content/en/docs/manual/advanced/slice-and-join.md

```diff
@@ -0,0 +1,67 @@
+---
+title: 'Join and slice tools'
+linkTitle: 'Join and slice tools'
+weight: 18
+description: 'This section explains how to slice or join several labels'
+---
+
+In CVAT you can modify shapes by either joining multiple shapes into
+a single label or slicing a single label into several shapes.
+
+This document provides guidance on how to perform these operations effectively.
+
+See:
+
+- [Joining CVAT labels](#joining-cvat-labels)
+- [Slicing CVAT labels](#slicing-cvat-labels)
+
+## Joining CVAT labels
```

Comment:
```suggestion
## Joining masks
```

Obviously we speak about CVAT, so, I do not really see a reason to add `CVAT`

Reply:
it's for google search